_jobs/ux-ui-designer-researcher.md

```diff
@@ -59,5 +59,5 @@ You’ll collaborate with a small team of designers and engineers. You’ll have
 
 ## To Apply
 
-Send resume, brief intro email and preferences for working (hours, time zone, etc) to [careers@compiler.la](mailto:careers@compiler.la) by July 1st 2022.
+Send resume, brief intro email and preferences for working (hours, time zone, etc) to [careers@compiler.la](mailto:careers@compiler.la) by April 12, 2024.
```

Comment:
Let's check with @shelby-e, I think we'll want to use our new ClickUp process for this instead of an email inbox.